AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_Default_SendReceiveSuccessfully.py

```diff
@@ -40,62 +50,40 @@ def ScriptEvents_Default_SendReceiveSuccessfully():
 
     :return: None
     """
-    import os
-    from editor_entity_utils import EditorEntity as Entity
-    from utils import Report
-    from utils import TestHelper as helper
-    from utils import Tracer
-
-    import azlmbr.legacy.general as general
-    import azlmbr.asset as asset
-    import azlmbr.math as math
-    import azlmbr.bus as bus
-
-    LEVEL_NAME = "tmp_level"
-    WAIT_TIME = 3.0  # SECONDS
-    EXPECTED_LINES = ["T92567320: Message Received"]
-    SC_ASSET_PATH = os.path.join("ScriptCanvas", "T92567320.scriptcanvas")
-
-    def create_editor_entity(name, sc_asset):
-        entity = Entity.create_editor_entity(name)
-        sc_comp = entity.add_component("Script Canvas")
-        asset_id = asset.AssetCatalogRequestBus(bus.Broadcast, "GetAssetIdByPath", sc_asset, math.Uuid(), False)
-        sc_comp.set_component_property_value("Script Canvas Asset|Script Canvas Asset", asset_id)
-        Report.critical_result(Tests.entity_created, entity.id.isValid())
-
-    def locate_expected_lines(line_list: list):
-        found_lines = [printInfo.message.strip() for printInfo in section_tracer.prints]
 
-        return all(line in found_lines for line in line_list)
+    def __init__(self):
+        editor_window = None
 
-    # 1) Create temp level
-    general.idle_enable(True)
-    result = general.create_level_no_prompt(LEVEL_NAME, 128, 1, 512, True)
-    Report.critical_result(Tests.level_created, result == 0)
-    helper.wait_for_condition(lambda: general.get_current_level_name() == LEVEL_NAME, WAIT_TIME)
-    general.close_pane("Error Report")
+    @pyside_utils.wrap_async
+    async def run_test(self):
 
-    # 2) Create test entity
-    create_editor_entity("TestEntity", SC_ASSET_PATH)
+        # Preconditions
+        general.idle_enable(True)
 
-    # 3) Start Tracer
-    with Tracer() as section_tracer:
+        # 1) Create temp level
+        hydra.open_base_level()
+        helper.wait_for_condition(lambda: general.get_current_level_name() == BASE_LEVEL_NAME, WAIT_TIME_3)
+        general.close_pane("Error Report")
 
-        # 4) Enter Game Mode
-        helper.enter_game_mode(Tests.enter_game_mode)
-
-        # 5) Read for line
-        lines_located = helper.wait_for_condition(lambda: locate_expected_lines(EXPECTED_LINES), WAIT_TIME)
-        Report.result(Tests.lines_found, lines_located)
+        # 2) Create test entity
+        entity = scripting_tools.create_entity_with_sc_component_asset("TestEntity", SC_ASSET_PATH)
+        helper.wait_for_condition(lambda: entity is not None, WAIT_TIME_3)
+        Report.critical_result(Tests.entity_created, entity.id.isValid())
+        
+        # 3) Start Tracer
+        with Tracer() as section_tracer:
```

Comment:
Does this really need it's own test step? This is a test environment action and not a test step itself.

Reply:
This is really more of a nitpick than a change request at this time.

If you happen to make any other changes, please group the tracer with the launching into game mode  step and update the test steps numbering and test description for both tests.

Reply:
removed this step and trimmed down the steps in the comment